com.unity.render-pipelines.high-definition/Editor/Material/BRGPicking.hlsl

```diff
@@ -0,0 +1,131 @@
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/UnityInstancing.hlsl"
+
+struct PickingAttributesMesh
+{
+    float3 positionOS   : POSITION;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
+
+struct PickingMeshToDS
+{
+    float3 positionRWS : INTERNALTESSPOS;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
+
+struct PickingMeshToPS
+{
+    float4 positionCS : SV_Position;
+    UNITY_VERTEX_INPUT_INSTANCE_ID
+};
+
+#ifdef TESSELLATION_ON
+#define PickingVertexOutput PickingMeshToDS
+#else
+#define PickingVertexOutput PickingMeshToPS
+#endif
+
+float4x4 unity_BRGPickingViewMatrix;
+float4x4 unity_BRGPickingProjMatrix;
+float4 unity_BRGPickingCameraWorldPos;
+
+#undef unity_ObjectToWorld
+float4x4 LoadObjectToWorldMatrixBRGPicking()
+{
+    float4x4 objectToWorld = LoadDOTSInstancedData_float4x4_from_float3x4(UNITY_DOTS_INSTANCED_METADATA_NAME(float3x4, unity_ObjectToWorld));
+#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING != 0)
+    objectToWorld._m03_m13_m23 -= unity_BRGPickingCameraWorldPos.xyz;
+ #endif
+    return objectToWorld;
+}
+
+float4 ComputePositionCS(float3 positionWS)
+{
+    float4x4 viewMatrix = unity_BRGPickingViewMatrix;
+    // HDRP expects no translation in the matrix because of camera relative rendering
+    viewMatrix._m03_m13_m23_m33 = float4(0,0,0,1);
```

Comment:
you need to test the define 
#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING != 0)
#endif

as it is possible to disable the camera relative in shader with shaderconfig.hlsl

Reply:
Solved by removing anything related to camera relative rendering. It's not needed at all anymore with an override picking shader (instead of fallback)